Userland/Libraries/LibWeb/CSS/CSSRuleList.cpp

```diff
@@ -141,6 +142,38 @@ void CSSRuleList::for_each_effective_style_rule(Function<void(CSSStyleRule const
         case CSSRule::Type::Supports:
             static_cast<CSSSupportsRule const&>(*rule).for_each_effective_style_rule(callback);
             break;
+        case CSSRule::Type::Keyframe:
+        case CSSRule::Type::Keyframes:
+            break;
```

Comment:
a `default` case with a debug log here too? 🤔

(also in other similar places, optionally: a `VERIFY`?)

Reply:
No. Not having a default makes the compiler tell you where you need to handle the new enum value.

Reply:
neat! (missed that feature!)